back/model/Artwork/V0.json

```diff
@@ -0,0 +1,23 @@
+{
+    "id": 1,
+    "name": "Allegoria de la Primavera",
+    "image": "https://upload.wikimedia.org/primavera",
+    "description": {
+        "fr": "Le Printemps (Primavera en italien prononcé : [primaˈvɛra]) est une peinture allégorique de Sandro Botticelli, exécutée à tempera sur panneau de bois entre 1478 et 1482, période de la Première Renaissance. Elle a été décrite comme « l'une des peintures les plus commentées et les plus controversées au monde », et aussi « l'une des peintures les plus populaires de l'art occidental »",
+        "en": "Primavera (Italian pronunciation: [primaˈvɛːra], meaning 'Spring'), is a large panel painting in tempera paint by the Italian Renaissance painter Sandro Botticelli made in the late 1470s or early 1480s (datings vary). It has been described as 'one of the most written about, and most controversial paintings in the world', and also 'one of the most popular paintings in Western art'"
+    },
+    "location": {
+        "type": "Point",
+        "coordinates": [40.01232, 50.02344],
+        "name": "Musée des Offices"
+    },
+    "artists": ["Sandro Botticelli"],
+    "style": {
+        "id": 1,
+        "name": {
+            "fr": "Peinture",
+            "en": "Painting"
+        }
+    },
+    "creationPeriod": "~1480"
```

Comment:
Why is creationPeriod a text field instead of a number or an object ?

Reply:
For some artwork, we don't know the exact year. In this case, you can have a large approximation or a year interval. I suggest so to put an text field which allows a lot of flexibility rather than a number.

Reply:
In that case maybe we should use a number, it would be easier to search in the database to have a date field or a number to "search by date", but it might not be useful in the first version of this project !

Reply:
We might have to create two fields: minDate and maxDate to search by date if we don't have an exact date for some artworks !